File: server/DocumentsKM/Models/DefaultValues.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DocumentsKM.Models
{
    // Значения по умолчанию
    public class DefaultValues
    {
        // Пользователь
        [ForeignKey("UserId")]
        public virtual User User { get; set; }
        [Key]
        public Int16 UserId { get; set; }

        // Отдел
        [ForeignKey("DepartmentId")]
        public virtual Department Department { get; set; }
        public Int16? DepartmentId { get; set; }

        // Разработчик
        [ForeignKey("CreatorId")]
        public virtual Employee Creator { get; set; }
        public Int32? CreatorId { get; set; }

        // Проверщик
        [ForeignKey("InspectorId")]
        public virtual Employee Inspector { get; set; }
        public Int32? InspectorId { get; set; }

        // Нормоконтролер
        [ForeignKey("NormContrId")]
        public virtual Employee NormContr { get; set; }
        public Int32? NormContrId { get; set; }
    }
}

File: client/src/model/DefaultValues.ts
import Department from './Department'
import Employee from './Employee'

interface IDefaultValues {
	department: Department
	creator: Employee
	inspector: Employee
	normContr: Employee
}

class DefaultValues {
	department: Department
	creator: Employee
	inspector: Employee
	normContr: Employee

	constructor(obj?: IDefaultValues) {
		this.department = (obj && obj.department) || null
		this.creator = (obj && obj.creator) || null
		this.inspector = (obj && obj.inspector) || null
		this.normContr = (obj && obj.normContr) || null
	}
}

export default DefaultValues
